python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -516,6 +519,17 @@ def run(self, sensorRef, templateIdList=None):
             mask  = subtractedExposure.getMaskedImage().getMask()
             mask &= ~(mask.getPlaneBitMask("DETECTED") | mask.getPlaneBitMask("DETECTED_NEGATIVE"))
 
+        # Compute ALZC correction kernel from matching kernel
+        # Here we use a constant kernel, just compute it for the center of the image.
+        # Investigating correction for spatially varying kernels is a subject of other tickets.
```

Comment:
I suggest removing this comment. The user can look up the definition of doDecorrelation above, and that's all you are doing here. Plus the log message is a clue. Details of the algorithm belong in the docs for "performExposureDecorrelation", since that may evolve over time, leaving the information here outdated.